tests/foreman/cli/test_computeresource.py

```diff
@@ -259,7 +259,7 @@ def test_create_cr_with_consolepwd(self):
 
     @tier1
     @run_only_on('sat')
-    def test_create_negative_1(self):
+    def test_negative_create(self):
```

Comment:
Actually here we are changing name, desc, url so we should name it `test_negative_create_with_name_description_url`